domain/plan_replayer.go

```diff
@@ -56,32 +56,37 @@ func parseTime(s string) (time.Time, error) {
 	return time.Unix(0, i), nil
 }
 
-func (p *planReplayer) planReplayerGC(t time.Duration) {
+func (p *dumpFileGcChecker) gcDumpFiles(t time.Duration) {
 	p.Lock()
 	defer p.Unlock()
-	path := GetPlanReplayerDirName()
+	for _, path := range p.paths {
+		p.gcDumpFilesByPath(path, t)
+	}
+}
+
+func (p *dumpFileGcChecker) gcDumpFilesByPath(path string, t time.Duration) {
 	files, err := ioutil.ReadDir(path)
 	if err != nil {
 		if !os.IsNotExist(err) {
 			logutil.BgLogger().Warn("[PlanReplayer] open plan replayer directory failed", zap.Error(err))
```

Comment:
[PlanReplayer] -> [gcDumpFilesByPath]

Here we'd better add the `path` in the warn log.